packages/web/src/common/store/cache/users/sagas.ts

```diff
@@ -362,21 +362,19 @@ export function* fetchUserSocials({
   }
   user = yield* call(waitForValue, getUser, { handle })
   if (!user) return
-  const socials = yield* call(
-    audiusBackendInstance.getSocialHandles,
-    user.handle
-  )
+  const socials = yield* call(audiusBackendInstance.getSocialHandles, user)
 
   yield* put(
     cacheActions.update(Kind.USERS, [
       {
         id: user.user_id,
         metadata: {
-          twitter_handle: socials.twitterHandle || null,
-          instagram_handle: socials.instagramHandle || null,
-          tiktok_handle: socials.tikTokHandle || null,
-          website: socials.website || null,
-          donation: socials.donation || null
+          twitter_handle: user.twitter_handle || socials.twitterHandle || null,
```

Comment:
Since I've seen it twice now, I'm wondering if we couldn't just merge the two objects with `Object.assign` or just spreading the result of `getSocialHandles` into the user so we don't have to do this double application of `||` for every property.